client/src/components/pages/Stats.js

```diff
@@ -13,8 +15,9 @@ export default function Stats({ tourney, user }) {
   const fetchData = async () => {
     const [tourneyModel, currentSelectedStage] = await getStage(tourney);
 
-    const [players, stageMaps, stageStats] = await Promise.all([
+    const [players, teams, stageMaps, stageStats] = await Promise.all([
       get("/api/players", { tourney }),
+      get("/api/teams", { tourney }),
```

Comment:
is it possible to skip over this call if the tourney doesn't have teams, e.g.
```
tourneyModel.teams ? get("/api/teams", { tourney }) : Promise.resolve([])
```

Reply:
cool